_posts/0000-01-02-norbiboy1.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
> {
"author":"Abraham Lincoln",
"quote":"No man has a good enough memory to be a successful liar."
}